src/main/scala/org/broadinstitute/dsde/rawls/ws/RawlsApiService.scala

```diff
@@ -25,12 +27,12 @@ class SwaggerService(override val apiVersion: String,
   (implicit val actorRefFactory: ActorRefFactory)
   extends SwaggerHttpService
 
-class RawlsApiServiceActor(swaggerService: SwaggerService) extends Actor with RootRawlsApiService {
+class RawlsApiServiceActor(swaggerService: SwaggerService, val workspaceServiceConstructor: () => WorkspaceService) extends Actor with RootRawlsApiService with WorkspaceApiService {
```

Comment:
You're right, I was thinking they defaulted to being var vs val in contrast to case class